Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.py

```diff
@@ -129,6 +129,8 @@ def update_incident(self, incident_id: int, status: Optional[str], assigned_to:
         """
         body = assign_params(status=status, assignedTo=assigned_to, classification=classification,
                              determination=determination, tags=tags, comment=comment)
+        if assigned_to == "":
```

Comment:
Add a comment to explain this part is a workaround for the `assign_params` removing empty values.